client/containers/App/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -103,7 +104,7 @@ const Breadcrumbs = (props: Props) => {
 				<Avatar
 					avatar={avatar}
 					initials={title[0]}
-					// @ts-expect-error ts-migrate(2322) FIXME: Type '{ avatar: any; initials: any; communityData:... Remove this comment to see the full error message
+					// @ts-expect-error FIXME: CommunityData does not exist on Avatar?
```

Comment:
I think i removed this error bc it worked at some point, then broke?

Anyway, this prop does not even exist on `Avatar`,  `Avatar` finds `communityData` through a provider, think this can be removed safely.